origin-discovery/src/apollo/app.js

```diff
@@ -27,12 +28,53 @@ const bundle = promBundle({
 })
 app.use(bundle)
 
+// how frequently featured/hidden listings list updates
```

Comment:
I'd recommend moving all this new logic into a separate file. maybe called info.js ?

Reply:
👍